src/Components/ErrorsNotify/ErrorsNotify.tsx

```
@@ -0,0 +1,42 @@
+import { useContext, useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { TodoContext } from '../TodoContext/TodoContext';
+
+export const ErrorsNotify = () => {
+  const { errorMessage, setErrorMessage } = useContext(TodoContext);
+  const [showErrorMessage, setShowErrorMessage] = useState(false);
+
+  useEffect(() => {
```

you should automatically hide the error after 3 seconds (read requirements)